### DISCUSSION TITLE
UX problem maybe?

### DISCUSSION THREAD

**Discussion by MuGi-131** [brianpetro/obsidian-smart-connections#764] [Q&A]
I use obsidian mainly because of its extensive keyboard customization and VIM support. I find this plugin severely lacking basic support on this department. Maybe it would be great if I after finish typing the prompt in the input field, at least push to [ENTER / RETURN] button to submit the prompt. My flow just gets disrupted by the time I transition to the MOUSE and searching and then clicking on tiny button.

tldr: Please add support for keyboard shortcuts.

**Comment by MuGi-131** [comment]
Similar issue has been raised.
